src/server.h

```diff
@@ -1176,6 +1176,8 @@ typedef struct client {
     size_t ref_block_pos;        /* Access position of referenced buffer block,
                                   * i.e. the next offset to send. */
 
+    /* list node in clients_pending_write list */
+    listNode clients_pending_write_node;
```

Comment:
Hi, may i ask why `clients_pending_read` is not doing this, it looks the we can do the same thing for it.

Reply:
The choice to do the clients_pending_write list node was based off of performance that probably have IO threading disabled. It would be easy enough to test the performance of it to see if it also benefits.